tests/test_decorator.py

```diff
@@ -485,7 +444,7 @@ def foo(dt: datetime = Field(default_factory=lambda: 946684800), /):
     assert module.foo(0) == datetime(1970, 1, 1, tzinfo=timezone.utc)
 
 
-@pytest.mark.xfail(reason='working on V2')
+@pytest.mark.xfail(reason='config["extra"] does not seem to be respected')
```

Comment:
This is kind of strange to me in the first place. If we think of the decorator as creating a model and the parameters as its fields, then why would the config (which is at the model level) apply to one (all?) of the fields?

I do think that in general we need to figure out a way to apply a config (or the portion of it that is applicable) to a `TypedDict` model.

Reply:
Yeah, I think probably if we are going to raise type errors for signature issues we should not support "extra" in the validate_arguments decorator. I'll admit I haven't ever used this decorator though, so I don't know if people have good reasons to want this sort of thing.